beluga_amcl/docs/PARAMETERS.md

```diff
@@ -0,0 +1,75 @@
+# Beluga AMCL Parameter Reference
+
+## ROS Interface Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `base_frame_id` | Robot base frame name rigidly attached to the mobile robot base. | ✅ | ✅ |
+| `odom_frame_id` | Odometry frame name. The pose of a mobile platform relative to this frame can drift over time but it must be continuous (without discrete jumps). | ✅ | ✅ |
+| `global_frame_id` | Map frame name. The pose of a mobile platform relative to this frame does not drift significantly over time but it can change in discrete jumps at any time. This node can estimate and publish the transform between global and odometry frames. | ✅ | ✅ |
+| `scan_topic` | The name of the topic where laser scans are being published. | ✅ | ✅ |
+| `map_topic` | The name of the topic where the map is published by the map server. | ✅ | ✅ |
+| `initial_pose_topic` | The name of the topic where an initial pose can be published. _Navigation 2 supports a fixed subscriber in the `initialpose` topic._ | | ✅ |
+
+## Initial Pose And Transform Broadcast Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `set_initial_pose` | Causes AMCL to set initial pose from the `initial_pose*` parameters instead of waiting for the initial pose message. | ✅ | ✅ |
+| `initial_pose.[x, y, yaw]` | X, Y and yaw coordinates of initial pose of robot base frame in global frame. Units in meters and radians. | ✅ | ✅ |
```

Comment:
@nahueespinosa nit: about units, we can simply say that both abide to [REP-103](https://www.ros.org/reps/rep-0103.html).

Reply:
Added an additional notes section and removed per-parameter unit specifications here https://github.com/Ekumen-OS/beluga/pull/196/commits/b0672a7827aa45034ea11d6fa2404de3df918006.